app/services/scene-collections/nodes/node-map.ts

```diff
@@ -0,0 +1,28 @@
+import { Node } from './node';
+import { Inject } from 'services/core';
+import { SceneCollectionsService } from 'services/scene-collections';
+
+interface ISchema {
+  sceneNodeMaps: { [sceneId: string]: Dictionary<string> };
+}
+
+export class NodeMapNode extends Node<ISchema, {}> {
+  schemaVersion = 1;
+
+  @Inject() sceneCollectionsService: SceneCollectionsService;
+
+  async save() {
+    const activeCollection = this.sceneCollectionsService.activeCollection;
+    if (activeCollection?.sceneNodeMaps) {
+      this.data = {
+        sceneNodeMaps: this.sceneCollectionsService?.sceneNodeMaps,
+      };
+    }
+  }
+
+  async load() {
+    if (this.data?.sceneNodeMaps && !this.sceneCollectionsService.hasOwnProperty('sceneNodeMaps')) {
```

Comment:
i'm a little worried about the way this conditional is set, i think it could lead to desync issues if a local scene collection has an outdated node map

Reply:
Yeah, that's a great point. Pushed up a suggested fix.